src/pages/de/core-concepts/framework-components.md

```diff
@@ -0,0 +1,234 @@
+
+Astro unterstützt eine Vielzahl von beliebten Frameworks wie [React](https://reactjs.org/), [Preact](https://preactjs.com/), [Svelte](https://svelte.dev/), [Vue](https://vuejs.org/), [SolidJS](https://www.solidjs.com/), [AlpineJS](https://alpinejs.dev/) und [Lit](https://lit.dev/).
+
+## Integrationen installieren
+
+Astro kommt mit optionalen Integrationen für React, Preact, Svelte, Vue, SolidJS, AlpineJS und Lit. Ein oder mehrere dieser Astro-Integrationen kann in deinem Projekt installiert und konfiguriert werden.
+
+Um Astro für die Verwendung dieser Frameworks zu konfigurieren, installiere zunächst die Integration mit ihren Abhängigkeiten:
+
+```bash
+npm install --save-dev @astrojs/react react react-dom
+```
+
+Anschließend importiere die Integration und füge die Funktion in deine Liste von Integrationen in der `astro.config.mjs` hinzu:
+
+```js title="astro.config.mjs" ins={3} ins=/(?<!p)react\\(\\)/
+import { defineConfig } from 'astro/config';
+
+import react from '@astrojs/react';
+import preact from '@astrojs/preact';
+import svelte from '@astrojs/svelte';
+import vue from '@astrojs/vue';
+import solid from '@astrojs/solid-js';
+import lit from '@astrojs/lit';
+import alpine from '@astrojs/alpinejs';
+
+export default defineConfig({
+	integrations: [react(), preact(), svelte(), vue(), solid(), lit(), alpine()],
+});
+```
+
+⚙️ Schaue den [Integrationsleitfaden](/en/guides/integrations-guide/) für mehr Details an und erfahre mehr über das Installieren und Konfigurieren von Astro-Integrationen.
+
+⚙️ Möchtest du ein Beispiel für ein Framework deiner Wahl sehen? Besuche [astro.new](https://astro.new/) und wähle eine Framework-Vorlage.
+
+## Framework-Komponenten nutzen
+
+Nutze deine JavaScript Framework-Komponenten in deinen Astro-Seiten, -Layouts und -Komponenten genauso wie deine Astro-Komponenten. Alle deine Komponenten können zusammen in `/src/components` organisiert werden, oder auf eine andere Weise, die dir gefällt.
+
+Um eine Framework-Komponente zu nutzen, importiere sie relativ in deine Astro-Komponente. Anschließend kannst du die Komponente neben anderen Komponenten, HTML-Elementen und JSX-ähnlichen Ausdrücken im Komponenten-Template verwenden.
+
+```astro title="src/pages/static-components.astro" ins={2,7}
+---
+import MyReactComponent from '../components/MyReactComponent.jsx';
+---
+<html>
+  <body>
+    <h1>Nutze React-Komponenten direkt in Astro!</h1>
+    <MyReactComponent />
+  </body>
+</html>
+```
+
+Standardmäßig werden die Framework-Komponenten als statisches HTML gerendert. Dies ist hilfreich für Template-Komponenten, die nicht interaktiv sind. Dadurch wird verhindert, dass unnötiges JavaScript an den Client gesendet wird.
+
+## Interaktive Komponenten hydratisieren
+
+Eine Framework-Komponenten kann kann aktiviert (hydratisiert) werden, indem die `client:*`-Direktiven verwendet werden. Das ist ein Komponenten-Attribut, welches definiert wie die Komponente **gerendert** und **hydriert** werden soll.
```

Comment:
It seems that this section has been updated in the English sources in the meanwhile. Could you please have a look at the English version and update the translation accordingly?

Reply:
Yes, I will check it - thanks for the hint.
Just thinking about how to deal with the problem that translations get out of sync when the source (en docs) have been changed 🤔